go_lib/dependency/dependency.go

```diff
@@ -60,7 +62,16 @@ func init() {
 
 // NewDependencyContainer creates new Dependency container with external clients
 func NewDependencyContainer() Container {
-	return &dependencyContainer{}
+	return &dependencyContainer{
+		helmClient: clients{
+			clients: make(map[string]helm.Client),
+		},
+	}
+}
+
+type clients struct {
+	clients map[string]helm.Client
+	m       sync.Mutex
```

Comment:
```suggestion
        m       sync.Mutex
	clients map[string]helm.Client
```

sure it doesn't matter. But it's a kind of rule of thumb, to put fields which mutex protects, under that mutex